server/views/pages/reviewer/view-statements.html

```diff
@@ -64,6 +61,13 @@ <h1 class="govuk-heading-xl">{{ pageTitle }}</h1>
                   }) 
                 }}                  
               {% endif %}
+
+              {% if not statement.isVerified and (removalRequested or statement.isOverdue) %}
```

Comment:
'Email not verified' link relocated below other links as it has longer text. Applied top margin spacing if there is more one item to display